README.md

```diff
@@ -7,7 +7,33 @@ By default, each TeamCity installation runs under a Professional Server license
 
 By pulling this image you accept the [JetBrains license agreement for TeamCity (Commercial License)] (https://www.jetbrains.com/teamcity/buy/license.html)
 
-This docker image can use (optionally) an external postgress database instead of the internal database of teamcity. (recommended for production usage)
+This docker image can use (optionally) an external postgress or mysql database instead of the internal database of TeamCity. (recommended for production usage)
+
+How to use this image?
+---------------
+
+A typical way to create the container and make sure it gets restarted:
+
+    docker run -v /var/lib/teamcity:/var/lib/teamcity -v /dev/urandom:/dev/random -p 8111:8111 --restart=always --name teamcity teamcity
+
+Note the `urandom` volume. Without it TeamCity can take up to 30 minutes to start due to waiting for bits to appear in `/dev/random`. (Don't worry, [it's okay](http://www.2uo.de/myths-about-urandom/).)
```

Comment:
I didn't have time to research what component was actually blocking on this read (I had already spent far too long staring at slow motion startups). It could be in TeamCity, it could be in Tomcat. 

I think the amount of entropy in the system I was testing on was ordinary (a Google Compute Engine Ubuntu 14.04 instance). If you destroy and recreate the container a number of times you will likely run into this as well. That's the nature of `/dev/random` -- it can easily run out.

It's your README though so happy to add any comment you like. You are correct that there are situations where it's not necessary. I just wouldn't accept the unpredictability myself. :)